src/aics-image-viewer/components/CellViewerCanvasWrapper/index.tsx

```diff
@@ -66,7 +87,7 @@ export default class ViewerWrapper extends React.Component {
   }
 }
 
-const STYLES = {
+const STYLES: { [key: string]: React.CSSProperties } = {
```

Comment:
This works great. I probably would have done
```
const STYLES = {
viewer: {...} as React.CSSProperties,
etc
};
```

Reply:
I preferred typing the overall object in part for completeness. It eliminates a (pretty trivial) extra hoop to jump through if e.g. someone needs to add a style to one of these objects later down the line, and from a readability perspective makes clear that the purpose of the object is only to hold collections of properties.